vkcs/internal/util/util.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/clients"
 )
 
+const RequestIDHeader = "X-Openstack-Request-Id"
```

Comment:
does it work with karboii and ghosts?

Reply:
These services do not send any request id, thus we do not handle it in corresponding resources